### ISSUE TITLE
TextBox doesn’t show keyboard

### ISSUE THREAD

**Issue by DasLixou** [vizia/vizia#304]
Was on my phone reading around the vizia repo and looked into the Demo. Then when I clicked the Textfeld, it was selected but the keyboard wouldn’t show up. I think this could also be a problem for PCs where it should show a touch keyboard.

**Comment by geom3trik** [comment]
Thank you for the report. You're right that on wasm the textbox does not yet support this. I suspect some javascript shenanigans is required but I'll have to do some research as js is not my forte.

**Comment by DasLixou** [comment]
Yeah and it may also be a problem for PCs with a touch keyboard when it runs native, so also without any js and wasm
